source/administration/config-settings.rst

```diff
@@ -613,25 +613,25 @@ ________
 
 AD/LDAP (Enterprise)
 ```````````````````````````
-Enable sign-in with LDAP
+Enable sign-in with AD/LDAP
```

Comment:
Yep, per https://github.com/mattermost/platform/pull/3925 and https://mattermost.atlassian.net/browse/PLT-3766